hnn_core/feed.py

```diff
@@ -179,6 +179,14 @@ def __create_extgauss(self):
     def __create_extinput(self):
         """Creates the ongoing external inputs (rhythmic)."""
         # print("__create_extinput")
+        # Escape if all synaptic weights are 0
+        for key in self.p_ext.keys():
+            if key.startswith('L2Pyr') or \
+                    key.startswith('L5Pyr') or \
+                    key.startswith('L2Bask') or \
+                    key.startswith('L2Bask'):
```

Comment:
should this be:

```suggestion
                    key.startswith('L5Bask'):
```
?